SECURITY.md

```diff
@@ -0,0 +1,145 @@
+- [Out of scope](#out-of-scope)
+- [Responsible Disclosure](#responsible-disclosure)
+- [PGP key to use if reporting via email](#pgp-key-to-use-if-reporting-via-email)
+
+## Supported Versions
+
+We currently support the latest stable release of the Tact compiler and infrastructure libraries.
+
+| Versions | Supported |
+| -------- | --------- |
+| Latest   | ✅ Yes    |
+| 1.6.x    | ✅ Yes    |
+| < 1.6    | ❌ No     |
+
+You can check the latest Tact compiler version at <https://www.npmjs.com/package/@tact-lang/compiler>.
+
+## Reporting Security Issues
+
+If you discover a security vulnerability in the Tact compiler, standard library, tooling, or related infrastructure, **please report it privately and responsibly**.
+
+### How to Report
+
+Please send us a detailed report using either email or the GitHub Security Advisory ["Report a Vulnerability"](https://github.com/tact-lang/tact/security/advisories/new) tab.
+
+📧 **Email:** `it+tact@tonstudio.io`
+🔒 Please encrypt your email using our [PGP key](#pgp-key-to-use-if-reporting-via-email).
+
+Your report should include:
+
+- A description of the vulnerability.
+- Steps to reproduce (if applicable).
+- The potential impact.
+- Any suggestions for remediation (if known).
+
+Please include all details necessary to reproduce the vulnerability, such as:
+
+- the input program that triggers the bug,
+- compiler version affected,
+- target TVM version,
+- the link to the mainnet or testnet transaction that demonstrates the bug,
+- if reproducing the bug using local emulators, such as [Sandbox](https://github.com/ton-org/sandbox), please add the test script, the instructions to install its dependencies, and the command to run the script to reproduce the issue,
+- the Tact language server version,
+- IDE or text editor, if applicable,
+- any relevant details about your execution environment, like the Node.js version or your operating system.
+
+Please include steps to reproduce the bug you have found in as much detail as possible.
+
+We will acknowledge your report within three business days and aim to provide a resolution or mitigation plan within 30 days. If a resolution requires more time, we will keep you updated on the progress.
+
+## In scope
+
+This policy applies to:
+
+- The Tact compiler, including the TypeScript wrappers it generates,
+- The Tact standard library (`stdlib`),
+- Developer tooling: Tact's CLI, VS Code plugin, and the Tact [language server](https://github.com/tact-lang/tact-language-server),
+- The Tact [documentation](https://docs.tact-lang.org),
+- Web-based compiler services, such as [TON Web IDE](https://ide.ton.org),
+- Smart contract generation and deployment utilities.
+
+A security issue, in this case, may stem from miscompilation, incorrect documentation description, incorrect language server, editor plugin, or TON Web IDE suggestions, which can lead to monetary loss if an affected smart contract is deployed in the mainnet and a malicious third party interacts with it.
```

Comment:
I would replace this with severity level descriptions. Attacks leading to DoS attacks, gas draining or similar issues are problematic but definitely not critical.

Reply:
Also, would be nice to have a bullet-list here as well.